code/modules/hydroponics/hydroponics.dm

```diff
@@ -1,5 +1,3 @@
-#define HYDRO_CYCLES_PER_AGE	2	//Adjust this to adjust how many hydroponics cycles it takes to increase age. Positive integers only.
```

Comment:
Change this to 1 (instead of 2) and leave the rest of the code. This would allow down streams to set things as they want while still accomplishing the same result you want

Reply:
So for a downstream, revert these changes and make the `2` a `1` to get the speed that was before. Am I reading that correctly?

Reply:
If this define is 1, then every hydro cycle the plant will age. This was the original speed after we switched from bay's xenobotany to TG's botany.

If this define is 2, then the plant will age every other cycle (2 cycles per age). This was roughly the speed we had during the period we used bay's xenobotany and is also what the current live code version is using. If you go into botany right after you read this comment (and this PR is still not merged), this is the rate you would experience.

A downstream could theoretically set this to whatever they want (as long as it is a positive integer) to make plants take as long as they want to age. Setting this define to 100, for example, would mean your plants grow at such a slow rate that a single seed will take most of a shift to run its full life (if not more).

Reply:
I understand now. That's strange though, before the Hydro refactor I remember plants growing slow enough that I could walk off for a bit and not worry about them. Also having downtime while they were growing. Now I can't keep my eyes off them or they'll be on empty nutrient and water with near half filled with weeds

Reply:
I believe nutrient/water drain consumption and weed/pest build-up numbers were tweaked somewhere along the line. Age really only affects maturity, production, and total lifespan times.